src/components/global/Footer.jsx

```diff
@@ -1,8 +1,208 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Other Library Imports
+import dayjs from 'dayjs';
+import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
+// Image Imports
+import roseLogo from '/assets/rose_logo.png';
+import logoBlob from '/assets/logoBlob.svg';
+
+// Legal links
+const legalLinks = [
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'About'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Code of Conduct'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    title: 'Contact Us'
+  },
+  {
+    href: 'https://www.codepdx.org/',
+    target: '_blank',
+    rel: 'noopener',
+    ml: 0.5,
+    text: `©${dayjs().year()}`,
+    title: 'CODE PDX'
+  }
+];
+
+// Social media links
+const socialMediaLinks = [
+  {
+    href: 'https://github.com/codeforpdx/',
+    icon: <FaGithub size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://www.linkedin.com/company/code-pdx/',
+    icon: <FaLinkedin size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://www.meetup.com/Code-for-PDX/',
+    icon: <FaMeetup size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  },
+  {
+    href: 'https://discord.gg/x6b573et',
+    icon: <FaDiscord size={45} />,
+    target: '_blank',
+    rel: 'noopener'
+  }
+];
+
+// renders socialMediaLinks for placement in stack below
+const renderSocialLinks = socialMediaLinks.map(({ href, icon }) => (
+  <Link
+    key={href}
+    href={href}
+    target="_blank"
+    rel="noopener"
+    color="#000"
+    sx={{
+      '&:hover': {
+        color: 'secondary.main'
```

Comment:
don't think this color flies on the grey, and most likely not on the white either. Not enough contrast to see it.

Reply:
they grey was just so I can keep track of where the container was for setting up viewport scaling and will be removed before the footer is complete. Do you have suggestions to change secondary.main to? <img width="278" alt="Screenshot 2023-10-16 at 10 30 37 AM" src="https://github.com/codeforpdx/codepdx_website/assets/114430929/c10d12e3-6360-4067-bc00-0658159de1e8">